tools/dynamo/verify_dynamo.py

```diff
@@ -215,9 +215,8 @@ def main():
         f"ROCM version: {rocm_ver}\n"
     )
     for args in _SANITY_CHECK_ARGS:
-        # TODO remove check when 3.11 is supported
-        if sys.version_info >= (3, 11):
-            warnings.warn("Dynamo not yet supported in Python 3.11. Skipping check.")
+        if sys.version_info >= (3, 12):
```

Comment:
@e should leave a warning for 3.11 until everything is working fine no?

Reply:
I'll leave a warning in eval_frame.py